pkg/extprom/http/metrics.go

```diff
@@ -29,26 +30,37 @@ func newDefaultMetrics(reg prometheus.Registerer, buckets []float64, extraLabels
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		requestSize: promauto.With(reg).NewSummaryVec(
 			prometheus.SummaryOpts{
 				Name: "http_request_size_bytes",
 				Help: "Tracks the size of HTTP requests.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		requestsTotal: promauto.With(reg).NewCounterVec(
 			prometheus.CounterOpts{
 				Name: "http_requests_total",
 				Help: "Tracks the number of HTTP requests.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		responseSize: promauto.With(reg).NewSummaryVec(
 			prometheus.SummaryOpts{
 				Name: "http_response_size_bytes",
 				Help: "Tracks the size of HTTP responses.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
+		inflightHTTPRequests: promauto.With(reg).NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "http_inflight_requests",
+				Help: "Current number of active requests.",
```

Comment:
maybe we could add a more specific description, something like:
```suggestion
				Help: "Current gauge of in-flight requests.",
```
wdyt?

Reply:
I think it's redundant to describe it as a "gauge" in the help, because counters should have the postfix `_total` as per [Prometheus' naming conventions](https://prometheus.io/docs/practices/naming/). If a metric's name doesn't end with `_total`, it should be or behave like a gauge.

To follow up, I don't think the help should say exactly the same as the metric name. Otherwise the help is not helpful. So I thought, "maybe someone will not know exactly what we mean with 'in-flight' and 'active' is a more commonly used word".

Reply:
I think when the metric ends with `_total` indeed it should indicate a counter, but if it does not end with `_total` not necessarily means that it is a gauge, as far as I understand (for [example](https://github.com/prometheus/prometheus/blob/01c55822169a9123367e333dc407edd8a263f75d/discovery/kubernetes/client_metrics.go))
I've seen other examples where also the name of the metric repeats its type in the help message (e.g. [here](https://promlabs.com/blog/2020/09/25/metric-types-in-prometheus-and-promql#histogram-metrics)) but not necessarily means it should be like that.

I think would be ok keeping `number` - though regarding `active` requests for me reads a bit more unclear, since I wouldn't know what the definition of an `active` request would be.

Reply:
Maybe it would be a good idea to define "in-flight" within the help message? 🙂

Reply:
@jessicalins the metrics in the reference you sent that aren't explicitly a gauge are summaries/histograms. They will though generate gauges/counters with the `_sum` and `_total` (respectively) postfixes under the hood.

I improved the help text for the in-flight HTTP requests metrics in 
7f0adc4. 👍